ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/zscaler_endpoints.schema.yml

```diff
@@ -11,13 +11,8 @@ keys:
     description: |-
       PREVIEW: These keys are in preview mode.
 
-      Special data model used for WAN internet-exit integration with Zscaler.
-      The model is supposed to be autofilled per-device using the `arista.avd.cv_zscaler_endpoints` Ansible lookup plugin.
-      Example:
-      ```yaml
-      zscaler_endpoints: "{{ lookup('arista.avd.cv_zscaler_endpoints', cv_server="<optional servername>", cv_token="<optional token>") }}"
-      ```
-      The cv_server and cv_token options can be set as inventory variables instead of given as arguments. (Then they will also be used for cv_deploy)
+      Special data model used for testing the WAN internet-exit integration with Zscaler.
+      The model is supposed to be autofilled per-device by `eos_designs`, but when CloudVision is not available, it is possible to provide the settings manually.
```

Comment:
```suggestion
      Special data model used for testing the WAN internet-exit integration with Zscaler.
      The model is supposed to be autofilled per-device by `eos_designs`.
      Manually setting this model will take precedence and prevent `eos_designs` from trying to contact CloudVision.
      This can be useful for offline testing or if CloudVision is not available or configured with Zscaler integration.
```